specification/resources/resource-manager/Microsoft.Authorization/2017-06-01-preview/examples/listPolicySetDefinitions.json

```diff
@@ -0,0 +1,67 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "api-version": "2017-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/providers/Microsoft.Authorization/policySetDefinitions/costManagement",
+            "type": "Microsoft.Authorization/policySetDefinitions",
+            "name": "costManagement",
+            "properties": {
+              "displayName": "VM and Storage Cost Management",
+              "description": "Policies required to minimize the risk of accidental cost overruns",
+              "metadata": {
+                "category": "Cost Management"
+              },
+              "parameters": {
+                "allowedLocations": {
+                  "type": "array",
+                  "metadata": {
+                    "description": "Regions that resources can be deployed to",
+                    "displayName": "Valid Regions"
+                  }
+                }
+              },
+              "policyDefinitions": [
+                {
+                  "policyDefinitionId": "/subscriptions/subid/providers/Microsoft.Authorization/policyDefinitions/storageSkus",
+                  "parameters": {
+                    "allowedLocations": {
+                      "value": [
```

Comment:
To make this clear and valid can you change this to "parameters": { "validLocations": { "value": "[parameters('allowedLocations')]" } }?